test/test-applications/integrations/Directory.Build.props

```diff
@@ -1,3 +0,0 @@
-<Project>
-  <Import Project="..\Directory.Build.props" />
```

Comment:
Note: looking for `Directory.Build.props` up the folder hierarchy is already the default behavior.